packages/suite/src/components/suite/WebUsbButton.tsx

```diff
@@ -1,16 +1,23 @@
 import TrezorConnect from '@trezor/connect';
-import { Button, ButtonProps } from '@trezor/components';
-import { Translation } from 'src/components/suite';
+import { ButtonProps, IconButton } from '@trezor/components';
+import { useIntl } from 'react-intl';
+import messages from 'src/support/messages';
 
-export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => (
-    <Button
-        {...props}
-        icon="SEARCH"
-        onClick={e => {
-            e.stopPropagation();
-            TrezorConnect.requestWebUSBDevice();
-        }}
-    >
-        <Translation id="TR_CHECK_FOR_DEVICES" />
-    </Button>
-);
+export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => {
```

Comment:
Please, rename this component, it's unclear what it does 🌀

Reply:
Good point, but I will do it in another PR, there are many occurences of this in the code. 
I suggest name `RefreshDevicesButton`